webui-src/app/rswebui.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable camelcase */
+/* eslint-disable no-prototype-builtins */
```

Comment:
Is this required? Don't think these rules are being used here.

Reply:
Yes, it doesn't require here. Now, i update the changes